packages/x-date-pickers-pro/src/internal/hooks/useStaticRangePicker/useStaticRangePicker.tsx

```diff
@@ -19,6 +19,7 @@ import {
 import { DateRange, RangePosition } from '../../models/range';
 
 const PickerStaticLayout = styled(PickersLayout)(({ theme }) => ({
+  position: 'relative',
```

Comment:
For this one I need your opinion

Right now to `am/pm` icons are rendered outside of the static pickers in the `StaticNextDateTimePicker` component.
I saw it when doing the demo.
If you think the change is risky, I can do it in the demo instead of in the actual components.

Related to #5149

cc @alexfauquette

Reply:
Alex is probably the one to better answer this question, but from my previous testing, this change introduced some issues in certain cases/conditions, unless they are no longer relevant due to layout/style change.
IMHO, we should prioritise including the mentioned PR/fix for v6 stable. 🤔

Reply:
Maybe better to put the relative to `PickersLayoutContentWrapper`. But the only way to properly do that is to put the relative to the clock itself

Reply:
For the PR, would you put it on `PickersLayoutContentWrapper` or would you just put it on the demo and wait for #5149 for the actual fix ?

Reply:
For this case I'd be either for waiting for a proper fix and NOT doing anything - keeping the issue intact even in demos.
Or like you suggested—change style in the demo only. But that's extra work, which will eventually be removed and would reduce our "internal push" with providing a proper positioning fix. 🙈 😆

Reply:
OK let's keep the problem intact :+1: